query/src/api/http/v1/status.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+use std::time::SystemTime;
+
+use poem::web::Data;
+use poem::web::Json;
+use poem::IntoResponse;
+use serde::Deserialize;
+use serde::Serialize;
+
+use crate::sessions::SessionManager;
+
+#[derive(Serialize, Deserialize, Eq, PartialEq, Debug)]
+pub struct Status {
+    pub running_queries_count: u64,
+    // secs since epoch
+    pub last_query_started_at: Option<u64>,
+    // secs since epoch
+    pub last_query_finished_at: Option<u64>,
```

Comment:
how about add a instance_started_at timestamp to record when the query instance started.

if a query instances started but not getting any query, the last_query_started_at and last_query_finished_at would both be empty, but it's also worth triggering the auto suspend operator

Reply:
ok

Reply:
done